home-data-migration/src/main/resources/db/changelog/0.0.1/Issue183.xml

```diff
@@ -5,126 +5,7 @@
         xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
                       http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
 
-    <changeSet id="Issue183.001" author="Ilya-Ross">
-        <insert  dbms="postgresql"
-                 schemaName="public"
-                 tableName="users">
```

Comment:
as I remember well we don't remove changeSet, we create new file where we will remove this information

Reply:
No, we will reset db and migrate again without test data